### ISSUE TITLE
Write better examples

### ISSUE THREAD

**Issue by green-coder** [green-coder/unity3d-ddp-client#4]
It would be nice to write some more impressive examples and post some relevant screenshots on the `README.md` file.

**Comment by brean** [comment]
I have a nice example, where you can move stuff around in your browser and it moves in Unity/UWP: [Geometry Position Unity 3D WebGL](https://github.com/brean/geometry_position_Unity3D_WebGL).
I plan to make a short video showing how it works with some more explaination, but I need to see when I find the time for that.

**Comment by green-coder** [comment]
Good idea. An animated gif shown at the top of the `README.md` would sure be eye-catchy. 👍 

If you don't have enough time to make something perfect, go for the compromise and save yourself some time. 1 visual example is already way better than none.

Adjustments can always be made later.
